test/liquidity-protection/LiquidityProtection.js

```diff
@@ -3802,6 +3426,565 @@ describe('LiquidityProtection', () => {
                     }
                 });
             });
+
+            /**
+             * the migration tests consist of 3 standard pools:
+             * - XXX/BNT
+             * - YYY/BNT
+             * - ETH/BNT
+             * the deployer of the pools adds an initial liquidity to each one of them
+             * each one of 3 providers then adds 9 positions of protected liquidity to each reserve token on each pool
+             * the protected liquidity amounts range between 1% - 9% of the available space on each side of each pool
+             * the primary test verifies that each provider can migrate 6 positions, 27 positions and 54 positions
+             * the verification process compares the state of the system before and after each migration request
+             *
+             * the state of the system includes the following balances:
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * |               | Liquidity Provider | V3 Bancor Vault | Liquidity Protection Wallet |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Gov Token     |          I         |        I        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | XXX Token     |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | YYY Token     |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | ETH "Token"   |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Network Token |          I         |        D        |              I              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * | Pool Token    |          I         |        I        |              D              |
+             * +---------------+--------------------+-----------------+-----------------------------+
+             * the ones marked I are expected to be identical before and after each migration request
```

Comment:
maybe just remove the I, replace the D with X and note that the ones that are marked with X are expected to change after a migration, while others aren't

Reply:
Done